proto/raft_serverpb.proto

```diff
@@ -59,6 +59,9 @@ message SnapshotMeta {
     // It should only be used for v2 to send snapshot to v1.
     // See https://github.com/pingcap/tiflash/issues/7568
     uint64 commit_index_hint = 7;
+    // Flag reflects whether the snapshot is persisted with plain files
+    // or sst files. `true` means using plain files.
+    bool use_plain_file = 8;
```

Comment:
How about adding it to `SnapshotCFFile`? It's more flexible, eg. index regions have little data in the default CF but a large amount of data in the write CF.

Reply:
Reasonable. And the TODO -- "dump the plain file based on the size of each column family respectively" can be implemented based on it.